CONTRIBUTING.md

```diff
@@ -225,6 +225,7 @@ are:
 * Although out of date, [Tom Lee's great blog article][tlgba] is very helpful
 * [rustaceans.org][ro] is helpful, but mostly dedicated to IRC
 * The [Rust Compiler Testing Docs][rctd]
+* For @bors, [this cheetsheat][cheetsheat] is helpful (Remember to replace `@homu` with `@bors` in the commands that you use.)
```

Comment:
Ahh yes, this is true. @christopherdumas mind sending a follow-up PR with the fix?